src/ClientHandle.cpp

```diff
@@ -2725,7 +2727,10 @@ void cClientHandle::SendEntityAnimation(const cEntity & a_Entity, char a_Animati
 
 void cClientHandle::SendPlayerAbilities()
 {
-	m_Protocol->SendPlayerAbilities();
+	if (m_Player != nullptr)
+	{
+		m_Protocol->SendPlayerAbilities();
```

Comment:
There is only one place in the entire codebase where `m_Player = nullptr` is possible and it was only introduced in #4248.  I think you should be fixing that directly:
https://github.com/cuberite/cuberite/blob/bdc8e0b83c5e3600a372397e1ee7a5c02c93117a/src/Entities/Player.cpp#L151-L152